docs/reference/ingest/search-inference-processing.asciidoc

```diff
@@ -123,50 +126,10 @@ These pipelines can also be viewed, edited, and deleted in Kibana via *Stack Man
 You may also use the <<ingest-apis,Ingest pipeline APIs>>.
 If you delete any of these pipelines outside of the *Content* UI in Kibana, make sure to edit the ML inference pipelines that reference them.
 
-[discrete#ingest-pipeline-search-inference-update-mapping]
```

Comment:
The "Mappings" step has been removed from the pipeline creation workflow in 8.12.